### ISSUE TITLE
Compatability With Firefox 36

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#140]
```
Operating system and version (Windows 7 - 64bit) :
.Net Framework version : VBA
Office name and version(32) :
Browser name and version : Firefox 36
SeleniumWrapper version : 1.0.23.0

What steps will reproduce the problem with a public website ?
1. driver.start "firefox"
2.
3.

What is the expected output? What do you see instead?

Browser opens but VBA app responds saying :

Error number -2146232832 
"Browser not started. Use the command start or startRemotely."

Please provide any additional information below.


```

Original issue reported on code.google.com by `s...@mcclellandyarr.co.uk` on 26 Feb 2015 at 1:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `florentbr` on 2 Mar 2015 at 12:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Same problem, selenium just stoped working.
See message attached.
```

Original comment by `pemif...@gmail.com` on 6 Mar 2015 at 12:15

Attachments:
- [Error.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-2/Error.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Upgraded to version 36.0.4
Selenium still doesn't work.
```

Original comment by `pemif...@gmail.com` on 30 Mar 2015 at 3:05

Attachments:
- [erro.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-3/erro.jpg)
- [erro1.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-3/erro1.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Run QuickTest.vbs with FF 36, this is what i got (see attachment)
```

Original comment by `pemif...@gmail.com` on 30 Mar 2015 at 4:45

Attachments:
- [erro2.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-4/erro2.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Run QuickTest.vbs with FF 37.0.1 and got the same error as FF 36
```

Original comment by `pemif...@gmail.com` on 6 Apr 2015 at 11:09

**Comment by GoogleCodeExporter** [comment]
```
Any news about FF 37 compatibility?
```

Original comment by `pemif...@gmail.com` on 8 Apr 2015 at 2:22

**Comment by GoogleCodeExporter** [comment]
```
I'm having the same issue
```

Original comment by `eyeswith...@gmail.com` on 14 Apr 2015 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
 Same Here. 
```

Original comment by `will...@hyperxmedia.com` on 28 Apr 2015 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Just installed FF 37.0.2
Got this errors, one from my Access application the other from QuickTest.vbs
```

Original comment by `pemif...@gmail.com` on 30 Apr 2015 at 10:17

Attachments:
- [erro.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-9/erro.jpg)
- [erro1.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-140/comment-9/erro1.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 2.0.2.0
http://florentbr.github.io/SeleniumBasic/
```

Original comment by `florentbr` on 22 Jul 2015 at 4:23
- Changed state: **Fixed**
